patch/0001-hwmon-lm75-add-support-for-PCT2075.patch

```diff
@@ -0,0 +1,115 @@
+From 90b3a0b833131831bf259e3506aaf9f29317aae9 Mon Sep 17 00:00:00 2001
```

Comment:
what is this commit id? should be https://github.com/torvalds/linux/commit/557c7ffa2f28, right?

Reply:
Yes. I first brought in patch/0001-hwmon-lm75-add-support-for-PCT2075.patch which is a backport of torvalds/linux@557c7ff. When I ran checkpatch.pl against this patch it complained about missing device tree bindings. So I added the patches for 985e225142cf , be889be7785d, and 5ac6badc5aa0 to remedy this. All of these commits are present in Linus's tree but searching for them doesn't appear to be working.